src/features/feedback/SearchFeedback.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <section v-if="isOpen">
+    <utrecht-heading model-value :level="2">Feedback</utrecht-heading>
+    <feedback-form
+      @cancelled="feedbackCancelled"
+      @saved="feedbackSaved"
+      :id="id"
+      :name="name"
+    ></feedback-form>
+  </section>
+  <menu v-else>
+    <button @click="openFeedbackForm" class="utrecht-button">
```

Comment:
volgens mij hoort er een `<li>` in een `<menu>`: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu

Reply:
done